.devcontainer/Dockerfile

```diff
@@ -34,4 +34,4 @@ ENV PATH=/venv/bin:$PATH
 
 # change this entrypoint if it is not the same as the repo
 ENTRYPOINT ["blueapi"]
-CMD ["worker"]
+CMD ["run"]
```

Comment:
Could we put this back to `worker`? More descriptive and saves having to change the docs